src/utils/generator.js

```diff
@@ -226,7 +226,14 @@ export class Generator {
     args: Array<Value>,
     kind?: string
   ): AbstractValue {
-    return this.derive(types, values, args, nodes => t.callExpression(createCallee(), nodes));
+    return this.derive(types, values, args, (nodes: any) => t.callExpression(createCallee(), nodes));
```

Comment:
Why the sad : any typing?

Reply:
I need to get back to this. It may have been a transient issue, or it may have to do with flow types for Babel.